### PR TITLE
#1151: Hitchhikers Guide to Projection selection

### PR DESCRIPTION
**Pull request by bresciamattia**

### CODE REVIEW

**Review by mamup11 — APPROVED** [review]

File: factcast-site/documentation-docsy/content/en/Usage/HitchhikersGuide/projection-selection.md [hunk 1]
```diff
@@ -0,0 +1,319 @@
+---
+title: "Hitchhiker's Guide To Projection Selection"
+weight: 10
+type: docs
+---
+
+{{% alert title="Preface" %}}
+
+This guide is targeting an audience that is already informed about the projection types offered by Factus, and about
+their basics.
```
[the remark this answers is not in the document]
Comment (bresciamattia) [reply factcast-site/documentation-docsy/content/en/Usage/HitchhikersGuide/projection-selection.md:10]: just basics :smile: as a noun, not an adjective. I can replace it with "concepts" if it is more clear

File: factcast-site/documentation-docsy/content/en/Usage/HitchhikersGuide/projection-selection.md [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+
+[Managed Projection documentation]({{< ref "/usage/factus/projections/types/managed-projection">}})
+
+Preface: considering a Managed Projection that has its state externalized in a shared database.
+
+- **Scalability**: a Managed Projection enables the application to effectively handle the lifecycle of the views,
+  allowing to adapt it to the expected workload. With a shared database, the derived views are uniformly accessible and
+  consistent among multiple instances of the projection.
+
+- **Performance**: whenever a projection is updated, the most recent events since the last update are fetched from the
```
[the remark this answers is not in the document]
Comment (bresciamattia) [reply factcast-site/documentation-docsy/content/en/Usage/HitchhikersGuide/projection-selection.md:152]: If I understand your suggestion, it is about frequency of the updates, right? Other than that it depends on the number of events that are "new" after the last query, and on the business logic complexity, as written after this line.

File: factcast-site/documentation-docsy/content/en/Usage/HitchhikersGuide/projection-selection.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mamup11) [reply factcast-site/documentation-docsy/content/en/Usage/HitchhikersGuide/projection-selection.md:10]: I guess concepts sounds more elegant to me, but that is just pickiness, it's ok also as it is, so I leave it up to you

Comment (bresciamattia) [inline factcast-site/documentation-docsy/content/en/Usage/HitchhikersGuide/projection-selection.md:10]:
```suggestion
their concepts.
```

**Pull request merged**